ant-cli/src/wallet/fs.rs

```diff
@@ -137,7 +137,15 @@ pub(crate) fn select_wallet_address() -> Result<String, Error> {
         0 => {
```

Comment:
Do we only want to use the ENV wallet when we have 0 local wallets? Shouldn't we always default to the ENV wallet if it exists?